uaclient/entitlements/esm.py

```diff
@@ -33,13 +34,29 @@ def disable(
             update_apt_and_motd_messages(self.cfg)
         return disable_performed, fail
 
+    def setup_unauthenticated_repo(self):
+        series = util.get_platform_info()["series"]
+        suite_prefix = series + "-" + self.apt_repo_name
```

Comment:
@orndorffgrant  I added this to get the correct suite name.

Reply:
Ah! shame on me for not looking closely enough.

Can you please change the format strings below to make this more clear:
```diff
-                "{series}-security".format(series=suite_prefix),
-                "{series}-updates".format(series=suite_prefix),
+                "{suite_prefix}-security".format(suite_prefix=suite_prefix),
+                "{suite_prefix}-updates".format(suite_prefix=suite_prefix),
```

Reply:
actually, can we do all of this in the format string e.g.
```
                "{series}-{service_name}-security".format(series=series, service_name=self.apt_repo_name),
```